src/Common/StringSearcher.h

```diff
@@ -569,8 +569,10 @@ class StringSearcher<true, ASCII> : public StringSearcherBase
     uint8_t first{};
 
 #ifdef __SSE4_1__
-    /// vector filled `first` for determining leftmost position of the first symbol
-    __m128i pattern;
+    uint8_t second{};
```

Comment:
`first` and `second` are too generic as names. Should have been `first_needle_character` and `second_needle_character`.

Reply:
Done.